admin-dev/themes/new-theme/scss/pages/_payment.scss

```diff
@@ -0,0 +1,48 @@
+.adminpayment {
+  .card {
+    .module-item-list {
+      .module-item-wrapper-list {
+        @include media-breakpoint-down(sm) {
+          position: relative;
+          padding-left: 5rem;
+
+          ul {
+            padding-left: 0;
+          }
+
+          .text-center {
+            // stylelint-disable-next-line
+            text-align: left !important;
+          }
```

Comment:
Really ?

Reply:
Yes really

There is a choice to do there :

- Do we use bootstrap responsive helper classes to text-align-left on mobile (So we separate the responsive to 2 context => Markup and CSS side
- Do we force it inside CSS side in order to have the responsive only on CSS side (more maintainable) ?

Reply:
I think it is always the same : Boostrap Helper Classes, else CSS !

Reply:
It has been done there then : https://github.com/PrestaShopCorp/ps_checkout/pull/661